### PR TITLE
pull request gateway

### PR DESCRIPTION
**Pull request by maorfr**
covers part of https://jira.coreos.com/browse/APPSRE-1148

this PR introduces a PR gateway - in case the repo to submit PRs to is not reachable, define `pullRequestGateway` to be `sqs` and the requests will be routed to a queue.

another integration will be created to read messages and submit PRs to app-interface.

### CODE REVIEW

File: utils/sqs_gateway.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+class SQSGatewayInitError(Exception):
+    pass
+
+
+class SQSGateway(object):
+    """Wrapper around SQS AWS SDK"""
+
+    def __init__(self):
+        access_key = os.environ['aws_access_key_id']
```
Comment (jmelis) [inline utils/sqs_gateway.py]: add decorator

File: utils/sqs_gateway.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            aws_secret_access_key=secret_key,
+            region_name=region_name,
+        )
+        self.sqs = session.client('sqs')
+        self.queue_url = queue_url
+
+    def send_message(self, body):
+        self.sqs.send_message(
+            QueueUrl=self.queue_url,
+            MessageBody=str(body)
```
Comment (jmelis) [inline utils/sqs_gateway.py]: change to json.dumps

File: utils/sqs_gateway.py (hunk 1 above)
Comment (maorfr) [reply utils/sqs_gateway.py]: will do

File: utils/sqs_gateway.py (hunk 2 above)
Comment (maorfr) [reply utils/sqs_gateway.py]: done

**Pull request merged**